i18n/ko.json

```diff
@@ -7354,5 +7354,217 @@
+  },
+  {
+    "id": "api.command_share.remote_id_invalid.error",
+    "translation": "보안 연결 ID가 잘못되었습니다: {{.Error}}"
+  },
+  {
+    "id": "api.command_share.remote_id.help",
+    "translation": "기존 보안 연결의 ID입니다. 보안 연결을 추가하려면 '보안 연결' 명령을 참조하십시오."
+  },
+  {
+    "id": "api.command_share.remote_already_invited",
+    "translation": "보안 연결이 이미 초대되었습니다."
+  },
+  {
+    "id": "api.command_share.permission_required",
+    "translation": "공유 채널을 관리하기 위해 `{{.Permission}}`권한이 필요하다."
+  },
+  {
+    "id": "api.command_share.not_shared_channel_unshare",
+    "translation": "공유되지 않은 채널은 공유를 취소할 수 없습니다."
+  },
+  {
+    "id": "api.command_share.no_remote_invited",
+    "translation": "이 채널에는 보안 연결이 초대되지 않았습니다."
+  },
+  {
+    "id": "api.command_share.name",
+    "translation": "공유 채널"
+  },
+  {
+    "id": "api.command_share.must_specify_valid_remote",
+    "translation": "초대할 올바른 보안 연결 ID를 지정해야 합니다."
+  },
+  {
+    "id": "api.command_share.missing_action",
+    "translation": "작업 누락. 사용 가능한 작업: {{.Actions}}"
+  },
+  {
+    "id": "api.command_share.invite_remote.help",
+    "translation": "외부 중요 인스턴스를 현재 공유 채널에 초대합니다"
+  },
+  {
+    "id": "api.command_share.invitation_sent",
+    "translation": "공유 채널 초대를 `{{.Name}} {{.SiteURL}}`에 보냈습니다."
+  },
+  {
+    "id": "api.command_share.invalid_value.error",
+    "translation": "'{{.Arg}}' 잘못 된 값: {{.Error}}"
+  },
+  {
+    "id": "api.command_share.hint",
+    "translation": "[작업]"
+  },
+  {
+    "id": "api.command_share.fetch_remote_status.error",
+    "translation": "보안 연결에 대한 상태를 가져올 수 없습니다: {{.Error}}."
+  },
+  {
+    "id": "api.command_share.fetch_remote.error",
+    "translation": "보안 연결을 가져오는 중 오류 발생: {{.Error}}"
+  },
+  {
+    "id": "api.command_share.desc",
+    "translation": "현재 채널을 외부 매터모스트 인스턴스와 공유합니다."
+  },
+  {
+    "id": "api.command_share.could_not_uninvite.error",
+    "translation": "`{{.RemoteId}}` 초대를 취소할 수 없습니다: {{.Error}}"
+  },
+  {
+    "id": "api.command_share.check_channel_exist.error",
+    "translation": "공유 채널을 확인하는 중 오류 발생: {{.Error}}"
+  },
+  {
+    "id": "api.command_share.channel_status_id",
+    "translation": "채널 ID `{{.ChannelId}}` 상태"
+  },
+  {
+    "id": "api.command_share.channel_status.help",
+    "translation": "이 공유 채널의 상태를 표시합니다"
+  },
+  {
+    "id": "api.command_share.channel_shared",
+    "translation": "이제 이 채널이 공유되었습니다."
+  },
+  {
+    "id": "api.command_share.channel_remote_id_not_exists",
+    "translation": "이 채널에 대한 공유 채널 보안 연결 `{{.RemoteId}}`이(가) 존재하지 않습니다."
+  },
+  {
+    "id": "api.command_share.channel_invite_not_home.error",
+    "translation": "다른 곳에서 시작된 공유 채널에는 보안 연결을 초대할 수 없습니다."
+  },
+  {
+    "id": "api.command_share.channel_invite.error",
+    "translation": "`{{.Name}}`을(를) 이 채널로 초대하는 동안 오류가 발생했습니다: {{.Error}}"
+  },
+  {
+    "id": "api.command_share.available_actions",
+    "translation": "사용 가능한 작업: {{.Actions}}"
+  },
+  {
+    "id": "api.command_remote.unknown_action",
+    "translation": "알 수 없는 작업 '{{.Action}'"
```

Comment:
looks like this is wrong, will close this pr and fix this in the weblate and then open a fresh one